modules/sandboxed-containers-preparing-openshift-cluster.adoc

```diff
@@ -2,34 +2,35 @@
-// To Do: Add link to installer provisioned infrastructure: ../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[Installer provisioned infrastructure]
+* Your cluster must be xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[installed on bare metal infrastructure] with {op-system-first} workers.
+* Your cluster must use xref:../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[installer-provisioned infrastructure].
 +
 [IMPORTANT]
 ====
 * {sandboxed-containers-first} only supports {op-system} worker nodes. RHEL 7 or RHEL  8 nodes are not supported.
 * Nested virtualization is not supported.
 ====
 
-[id="sandboxed-containers-additional-resource-requirements_{context}"]
-== Additional resource requirements for {sandboxed-containers-first}
+[id="sandboxed-containers-resource-requirements_{context}"]
+== Resource requirements for {sandboxed-containers-first}
 
-{sandboxed-containers-first} is a product that brings the ability to run workloads inside a sandboxed runtime, such as Kata Containers, to your {product-title} clusters. Each pod is represented by a virtual machine (VM). Each VM runs in a `qemu` process and hosts a `kata-agent` process that acts as a supervisor for managing container workloads and processes that are running in those containers. There are two additional processes that add more overhead:
+{sandboxed-containers-first} lets users run workloads on their {product-title} clusters inside a sandboxed runtime (Kata). Each pod is represented by a virtual machine (VM). Each VM runs in a QEMU process and hosts a `kata-agent` process that acts as a supervisor for managing container workloads, and the processes running in those containers. Two additional processes add more overhead:
 
 * `containerd-shim-kata-v2` is used to communicate with the pod.
 * `virtiofsd` handles host file system access on behalf of the guest.
 
 Each VM is configured with a default amount of memory. Additional memory is hot-plugged into the VM for containers that explicitly request memory.
 
-* If a container runs without a given memory resource, it is able to consume free memory. It will do so until the total memory used by the VM reaches the default allocation. The guest and its I/O buffers also consume memory.
-* If a container is given a specific amount of memory, then that memory is hot-plugged into the VM before the container starts.
-* If a memory limit is specified, then the workload is terminated if it consumes more memory than the limit. If no memory limit is specified, the kernel  that is running on the virtual machine might run out of memory. If the kernel runs out of memory it might terminate other processes on the virtual machine.
+If a container runs without a memory resource, it will consume free memory until the total memory used by the VM reaches the default allocation. The guest and its I/O buffers also consume memory.
```

Comment:
Avoid using "it"

Reply:
How's this instead?

A container running without a memory resource consumes free memory until the total memory used by the VM reaches the default allocation. The guest and its I/O buffers also consume memory.